lib/daru/accessors/mdarray_wrapper.rb

```diff
@@ -1,9 +1,7 @@
 module Daru
   module Accessors
-
```

Comment:
Lets just remove this file. MDArray is not being maintained anymore. jNMatrix will be the future for JRuby arrays.